lib/bolero-engine/src/rng.rs

```diff
@@ -20,10 +23,18 @@ impl Default for Options {
 }
 
 impl Options {
+    pub fn test_time_or_default(&self) -> Duration {
+        self.test_time.unwrap_or_else(|| {
+            if self.iterations.is_some() {
+                Duration::from_secs(3600 * 24 * 365 * 100)
```

Comment:
Are you just trying to compute a large duration? Why not use https://doc.rust-lang.org/std/time/struct.Duration.html#associatedconstant.MAX?

Reply:
Nice catch! I hadn't thought of checking the docs for one such constant, this is now fixed :)